README.md

```diff
@@ -4,28 +4,40 @@ ChitChat is a Python-powered Social Network.
 
 ## Installation
 
-**TODO** Update the sections below:
+1. cd into backend folder
+    cd backend
+
+2. create virtual environment
+    ### Installing virtualenv
+    ```pip install virtualenv```
```

Comment:
Here you're installing the third party`virtualenv` but you don't use it, making it redundant. Let's remove this line.

Python now has a built-in `venv` module for managing virtual environments and that's exactly what you used below when you did ```python -m venv venv```. The -m flag tells Python to use a built in module.

Please remove the ```pip install virtualenv``` line.

Reply:
Noted.